lib/taurus/qt/qtgui/taurusgui/res/setup.template

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+"""
+This file was autogenerated by taurusgui --new-gui.
+"""
+
+from setuptools import setup, find_packages
+
+# version (bumpversion style)
```

Comment:
I don't know. For the one hand it does not disturb and for the other hand, I think is the developer who has to take the decision to use `bumpversion` and how to use it. 
Take in mind that the generated project are not under a git repo (Shall we do it?) 

I would not do more project configuration.